### ISSUE TITLE
Problem building Bioconductor extension lpsymphony on Fedora 32

### ISSUE THREAD

**Issue by verdurin** [easybuilders/easybuild-easyconfigs#11144]
I see this error when building `R-bundle-Bioconductor/3.11-foss-2020a-R-4.0.0`:
```
make[4]: Entering directory '/home/<user>/.local/easybuild/build/RbundleBioconductor/3.11/foss-2020a-R-4.0.0/lpsymphony/lpsymphony/src/SYMPHONY/Osi/src/Osi'
/bin/sh ../../../libtool --tag=CXX --mode=compile mpicxx -DHAVE_CONFIG_H -I. -I`echo .`  -I/tmp/Rtmp0boicg/Rbuild8d44a5f2a1f/lpsymphony/src/SYMPHONY/include/coin  -I/home/<user>/.local/easybuild/software/FFTW/3.3.8-gompi-2020a/include -I/home/<user>/.local/easybuild/software/ScaLAPACK/2.1.0-gompi-2020a/include -I/home/<user>/.local/easybuild/software/OpenBLAS/0.3.9-GCC-9.3.0/include  -w -g -O2   -DOSI_BUILD -c -o OsiAuxInfo.lo OsiAuxInfo.cpp
 mpicxx -DHAVE_CONFIG_H -I. -I. -I/tmp/Rtmp0boicg/Rbuild8d44a5f2a1f/lpsymphony/src/SYMPHONY/include/coin -I/home/<user>/.local/easybuild/software/FFTW/3.3.8-gompi-2020a/include -I/home/<user>/.local/easybuild/software/ScaLAPACK/2.1.0-gompi-2020a/include -I/home/<user>/.local/easybuild/software/OpenBLAS/0.3.9-GCC-9.3.0/include -w -g -O2 -DOSI_BUILD -c OsiAuxInfo.cpp  -fPIC -DPIC -o OsiAuxInfo.o
OsiAuxInfo.cpp:10:10: fatal error: CoinPragma.hpp: No such file or directory
   10 | #include "CoinPragma.hpp"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [Makefile:535: OsiAuxInfo.lo] Error 1
make[4]: Leaving directory '/home/<user>/.local/easybuild/build/RbundleBioconductor/3.11/foss-2020a-R-4.0.0/lpsymphony/lpsymphony/src/SYMPHONY/Osi/src/Osi'
make[3]: *** [Makefile:407: all] Error 2
make[3]: Leaving directory '/home/<user>/.local/easybuild/build/RbundleBioconductor/3.11/foss-2020a-R-4.0.0/lpsymphony/lpsymphony/src/SYMPHONY/Osi/src/Osi'
make[2]: *** [Makefile:532: all-recursive] Error 1
make[2]: Leaving directory '/home/<user>/.local/easybuild/build/RbundleBioconductor/3.11/foss-2020a-R-4.0.0/lpsymphony/lpsymphony/src/SYMPHONY/Osi'
make[1]: *** [Makefile:324: all-recursive] Error 1
make[1]: Leaving directory '/home/<user>/.local/easybuild/build/RbundleBioconductor/3.11/foss-2020a-R-4.0.0/lpsymphony/lpsymphony/src/SYMPHONY'
make: *** [Makevars:13: SYMPHONY.ts] Error 2
ERROR: compilation failed for package lpsymphony
```

**Comment by edmondac** [comment]
I've just seen this too, in EL8 on haswell.

**Comment by edmondac** [comment]
Maybe related to pkg-config... And https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.11-foss-2020a-R-4.0.0.eb has been updated since my attempt so I'll try the new version.

**Comment by edmondac** [comment]
Yep - it's succeeded for me now with the new version

**Comment by boegel** [comment]
I ran into this recently when I had `pkg-config` included in `--filter-deps`, while not having `pkg-config` available in the OS.

This issue is indeed caused by not having `pkg-config` available, it was fixed in #11104, so I'll close this.
